pkg/kafka/config.go

```diff
@@ -103,6 +110,12 @@ func (cfg *Config) Validate() error {
 	if cfg.ProducerMaxRecordSizeBytes < minProducerRecordDataBytesLimit || cfg.ProducerMaxRecordSizeBytes > maxProducerRecordDataBytesLimit {
 		return ErrInvalidProducerMaxRecordSizeBytes
 	}
+	if (cfg.TargetConsumerLagAtStartup != 0) != (cfg.MaxConsumerLagAtStartup != 0) {
```

Comment:
This condition looks weird to me. We are saying if both are different than 0 AND different from each other to return ErrInconsistentConsumerLagAtStartup?

Reply:
This is checking that either both values are set or both values are unset. An slightly more verbose alternative would be 
`if (cfg.TargetConsumerLagAtStartup != 0 && cfg.MaxConsumerLagAtStartup == 0) || (cfg.TargetConsumerLagAtStartup == 0 && cfg.MaxConsumerLagAtStartup != 0)`

Reply:
FYI, George raised the same thing below so I replaced the `(cfg.TargetConsumerLagAtStartup != 0) != (cfg.MaxConsumerLagAtStartup != 0)` logic with the clearer one I posted above.